verifier/src/tcb.rs

```diff
@@ -25,11 +49,75 @@ impl From<serde_json::Error> for Error {
     }
 }
 
-/// Raw unverified representation of the TCB info provided from
+/// The `tcb_info` member of the TCB(Trusted Computing Base) data retrieved from
+/// <https://api.trustedservices.intel.com/sgx/certification/v4/tcb?fmspc={}>
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TcbInfo {
+    id: String,
+    version: u32,
+    issue_date: String,
+    next_update: String,
+    fmspc: String,
+    pce_id: String,
+    tcb_type: u32,
```

Comment:
Still trying to get to the full schema but intel's website has been a bit flaky today. I'm pretty sure I've seen a schema for what is in the json, for now I mainly based this off the example.json and the _live_ one I pulled